test/e2e_mc/multicluster_test.go

```diff
@@ -0,0 +1,152 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e_mc
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+
+	"antrea.io/theia/test/e2e"
+)
+
+const (
+	iperfTimeSec = 12
+	// Set target bandwidth(bits/sec) of iPerf traffic to a relatively small value
+	// (default unlimited for TCP), to reduce the variances caused by network performance
+	// during 12s, and make the throughput test more stable.
+	iperfBandwidth = "10m"
+)
+
+var expectedNumDataRecords = 3
+
+func TestMultiCluster(t *testing.T) {
+	testData, _, v6Enabled, err := setupTestForMultiCluster(t)
+	if err != nil {
+		t.Errorf("Error when setting up test: %v", err)
+		failOnError(err, t, testData)
+	}
+	defer flowVisibilityCleanup(t, testData)
+	eastClusterTD := testData.clusterTestDataMap[eastCluster]
+	westClusterTD := testData.clusterTestDataMap[westCluster]
+	t.Logf("Creating Perftest Pods")
+	podAIPs, podBIPs, podCIPs, podDIPs, err := createPerftestPods(testData)
+	if err != nil {
+		failOnError(fmt.Errorf("error when creating perftest Pods: %v", err), t, testData)
+	}
+	t.Logf("Sending traffic in east cluster")
+	var cmdStr string
+	if !v6Enabled {
+		cmdStr = fmt.Sprintf("iperf3 -c %s -t %d -b %s", podBIPs.GetIPv4(), iperfTimeSec, iperfBandwidth)
+	} else {
+		cmdStr = fmt.Sprintf("iperf3 -6 -c %s -t %d -b %s", podBIPs.GetIPv6(), iperfTimeSec, iperfBandwidth)
+	}
+	_, _, err = eastClusterTD.RunCommandFromPod(multiClusterTestNamespace, "perftest-a", "perftool", []string{"bash", "-c", cmdStr})
+	require.NoErrorf(t, err, "Error when running iperf3 client: %v", err)
+	var clickHouseRecordsE []*e2e.ClickHouseFullRow
+	if !v6Enabled {
+		clickHouseRecordsE = e2e.GetClickHouseOutput(t, eastClusterTD, podAIPs.GetIPv4().String(), podBIPs.GetIPv4().String(), "", false, true)
+	} else {
+		clickHouseRecordsE = e2e.GetClickHouseOutput(t, eastClusterTD, podAIPs.GetIPv6().String(), podBIPs.GetIPv6().String(), "", false, true)
+	}
+	t.Logf("Verifying records in ClickHouse")
+	require.GreaterOrEqualf(t, len(clickHouseRecordsE), expectedNumDataRecords, "ClickHouse should receive expected number of flow records. Considered records: %v", clickHouseRecordsE)
+	t.Logf("Sending traffic in west cluster")
+	if !v6Enabled {
+		cmdStr = fmt.Sprintf("iperf3 -c %s -t %d -b %s", podDIPs.GetIPv4(), iperfTimeSec, iperfBandwidth)
+	} else {
+		cmdStr = fmt.Sprintf("iperf3 -6 -c %s -t %d -b %s", podDIPs.GetIPv6(), iperfTimeSec, iperfBandwidth)
+	}
+	_, _, err = westClusterTD.RunCommandFromPod(multiClusterTestNamespace, "perftest-c", "perftool", []string{"bash", "-c", cmdStr})
+	require.NoErrorf(t, err, "Error when running iperf3 client: %v", err)
+	var clickHouseRecordsW []*e2e.ClickHouseFullRow
+	if !v6Enabled {
+		clickHouseRecordsW = e2e.GetClickHouseOutput(t, eastClusterTD, podCIPs.GetIPv4().String(), podDIPs.GetIPv4().String(), "", false, true)
```

Comment:
If I understand correctly, we should use `westClusterTD` here instead of `eastClusterTD` to check flow records on west cluste?